### ISSUE TITLE
docker module: Binding host volume to volume not listed in dockerfile fails

### ISSUE THREAD

**Issue by geauxvirtual** [ansible/ansible-modules-core#148]
##### Issue Type:

Bug Report
##### Ansible Version:

1.8
##### Environment:

CentOS 6.5
docker 1.1.2
##### Summary:

When mounting a host volume to a container that is run from an image that did not list that volume as a mount point, the container does not successfully see the host volume. For example, the default nginx container does not list /etc/nginx/ssl as a mount point in the docker file. https://github.com/nginxinc/docker-nginx/blob/3713a0157083eb4776e71f5a5aef4b2a5bc03ab1/Dockerfile 

When starting a container based off the nginx image with docker run and passing -v /etc/nginx/ssl:/etc/nginx/ssl the container starts as expected. When trying this through Ansible, the container fails to start because the nginx service can not locate the SSL certificates. The only difference I could see when doing an inspect of these two containers is that the container started through Ansible added the /etc/nginx/ssl directory to the volumes under config, whereas the container started via docker run does not update the listed volumes in the config. This can be seen here in this gist showing the inspect output of two containers, one started with docker run the other with ansible. https://gist.github.com/geauxvirtual/eb1f37dfd2318234b497

Commenting out this line in the code, https://github.com/ansible/ansible-modules-core/blob/devel/cloud/docker.py#L389, the container starts and see the host mounted directory. 

I will try to replicate this on a newer version of docker as well.
##### Steps To Reproduce:

Add a host mounted volume to a container run from an image built from a dockerfile that did not list the volume as a mount point.
##### Expected Results:

Container should have access to the host mounted volume.
##### Actual Results:

Container can not see the host mounted volume although the host config shows the host mounted directory is mounted to the container directory.

**Comment by ansibot** [comment]
@cove, @softzilla, @zfil, @ThomasSteinbach, ping. This issue is waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@cove, @softzilla, @zfil, @ThomasSteinbach, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by zfil** [comment]
wontfix
this module is deprecated
@cove, @softzilla, @zfil, @ThomasSteinbach guys, in general I suggest we stop docker modules needing docker-py to rely only on native docker component (like compose)
